test/functional/Expression/AbstractExpressionTest.php

```diff
@@ -83,13 +82,13 @@ public function testGetTerms()
     /**
      * Tests the protected term setter method.
      *
-     * @since [*next-version*]
+     * @since 0.1
      */
     public function testSetTerms()
```

Comment:
This tests exactly what I said not to test. Don't test properties. Only the methods.